src/components/DiscoveryList.jsx

```diff
@@ -1,13 +1,77 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { introJs } from 'intro.js'
```

Comment:
Is`intro.js` a new package? I think you need to update the `package.json` and the `yarn.lock`, isn't it?